autoPlay.user.js

```diff
@@ -258,9 +258,9 @@ function MainLoop() {
 
 			if( s().m_rgStoredCrits.length > 0 )
 			{
-				var rgDamage = s().m_rgStoredCrits.splice(0,1);
+				var rgDamage = s().m_rgStoredCrits.splice(0,s().m_rgStoredCrits.length).reduce(function(a,b){return a + b});
```

Comment:
There's no reason to call splice here, just reduce it directly, and reset m_rgStoredCrits to an empty array.